### DISCUSSION TITLE
does anyone get meaningful results in stock price predict by rsfresh?

### DISCUSSION THREAD

**Discussion by nickhuangxinyu** [blue-yonder/tsfresh#842] [Q&A]
I must say tsfresh is fantastic.

but for the exciting example in apple stock price, can this extract features helps on prediction?

**Comment by nils-braun** [comment]
Thanks for the praise!

But I guess if stock market predictions would be so "easy" we would all be probably rich ;-)

Maybe someone can share their successes, but I fear that real-world stock market predictions might maybe use tsfresh as additional input, but be not based solely on that
